src/Megamix/Commands.cpp

```diff
@@ -0,0 +1,24 @@
+#include <3ds.h>
+#include <CTRPluginFramework.hpp>
+
+#include "Megamix.hpp"
+
+namespace Megamix{
+
+    extern "C" __attribute__((used)) void tickflowCommandsHook(CTickflow* self, u32 cmd_num, u32 arg0, u32* args){
```

Comment:
might be a good idea to split the `extern "C" __attribute__((used))` into a separate line for readability?